src/KeysInterface.php

```diff
@@ -0,0 +1,323 @@
+<?php
+
+namespace duncan3dc\PhpIni;
+
+/**
+ * Interface KeysInterface
+ *
+ * @package duncan3dc\PhpIni
+ * @author  Jawira Portugal <dev@tugal.be>
+ */
+interface KeysInterface
+{
+    const ALLOW_URL_FOPEN                       = 'allow_url_fopen';
+    const ALLOW_URL_INCLUDE                     = 'allow_url_include';
+    const APC_COREDUMP_UNMAP                    = 'apc.coredump_unmap';
```

Comment:
Hi @jawira this is a nice idea, but what do you think about putting them in namespaces and classes for each section, eg:
```php
\duncan3dc\PhpIni\Settings::ALLOW_URL_FOPEN;
\duncan3dc\PhpIni\Settings::ALLOW_URL_INCLUDE;
\duncan3dc\PhpIni\Settings\Apc::COREDUMP_UNMAP;
```
Or perhaps as namespaced constants autoloaded via a file:
```php
\duncan3dc\PhpIni\Settings\ALLOW_URL_FOPEN;
\duncan3dc\PhpIni\Settings\ALLOW_URL_INCLUDE;
\duncan3dc\PhpIni\Settings\Apc\COREDUMP_UNMAP;
```